backend/src/test/java/net/vidux/camhub/discovery/CameraFactoryTest.java

```diff
@@ -0,0 +1,7 @@
+package net.vidux.camhub.discovery;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CameraFactoryTest {
+  //TODO Need live data
+}
```

Comment:
Ez itt még érkezik, csak felküldtem, hogy a kód többi részét addig is lehessen nézni